contracts/interfaces/IBondingMinter.sol

```diff
@@ -0,0 +1,12 @@
+//SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+interface IBondingMinter {
+    event StartBonding(uint startTimestamp, uint endTimestamp);
```

Comment:
BondIssue

also we probably need the sum here

Reply:
> BondIssue
> 
> also we probably need the sum here

What do you mean by "sum"?